dev-haskell/hackage-security/hackage-security-0.6.0.1.ebuild

```diff
@@ -0,0 +1,58 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+# ebuild generated by hackport 0.6.5.9999
+
+CABAL_FEATURES="lib profile haddock hoogle hscolour test-suite"
+inherit haskell-cabal
+
+DESCRIPTION="Hackage security library"
+HOMEPAGE="https://github.com/haskell/hackage-security"
+SRC_URI="https://hackage.haskell.org/package/${P}/${P}.tar.gz"
+
+LICENSE="BSD"
+SLOT="0/${PV}"
+KEYWORDS="~amd64 ~x86"
+IUSE="mtl21 +network-uri"
```

Comment:
Same thing here as in `cabal-install-3.2.0.0`, only this time CI is complaining about `mtl21` being an invalid IUSE. That can be safely disabled, too.